org.torproject.torbrowser-launcher.yaml

```diff
@@ -1,15 +1,15 @@
 app-id: org.torproject.torbrowser-launcher
 runtime: org.kde.Platform
-runtime-version: 5.15-23.08
+runtime-version: 6.8
 sdk: org.kde.Sdk
 base: com.riverbankcomputing.PyQt.BaseApp
-base-version: 5.15-23.08
+base-version: 6.7
 cleanup-commands:
   - /app/cleanup-BaseApp.sh
 build-options:
   env:
     - BASEAPP_REMOVE_WEBENGINE=1
-separate-locales: false
+separate-locales: false 
```

Comment:
Please remove the extra space character at the end. It is harmless, but triggers my OCD. :-)) Thanks!

Reply:
Yeah nowadays, I get a lot of people who have that. :)